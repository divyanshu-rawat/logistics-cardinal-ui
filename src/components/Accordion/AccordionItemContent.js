import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

export const AccordionItemContentStyled = styled('div')`
  padding: 0 0 20px 0;
`;

const AccordionItemContent = ({ children }) => (
  <AccordionItemContentStyled>{children}</AccordionItemContentStyled>
);

AccordionItemContent.propTypes = {
  children: PropTypes.node.isRequired,
};

export default AccordionItemContent;
