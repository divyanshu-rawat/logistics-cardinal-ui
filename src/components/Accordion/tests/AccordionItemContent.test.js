import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import AccordionItemContent from '../AccordionItemContent';

const renderedComponent = () =>
  shallow(
    <AccordionItemContent>
      <p>test...</p>
    </AccordionItemContent>,
  );

describe('AccordionItemContent', () => {
  it('should render the component', () => {
    const renderComponent = renderedComponent();
    expect(toJson(renderComponent)).toMatchSnapshot();
  });
});
