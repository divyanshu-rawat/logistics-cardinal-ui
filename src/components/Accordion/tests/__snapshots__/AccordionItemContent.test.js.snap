// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccordionItemContent should render the component 1`] = `
<AccordionItemContent__AccordionItemContentStyled>
  <p>
    test...
  </p>
</AccordionItemContent__AccordionItemContentStyled>
`;
